OWNERS.md

```diff
@@ -0,0 +1,18 @@
+# Senior committers
+
+When possible, all PRs should be reviewed by at least one senior committer prior to merging.
+
+* Matt Klein (mattklein123) (mklein@lyft.com)
+* Harvey Tuch (htuch) (htuch@google.com) (pending)
+
+# Committers
+
+* Roman Dzhabarov (RomanDzhabarov) (rdzhabarov@lyft.com)
+* Constance Caramanolis (ccaraman) (ccaramanolis@lyft.com)
+* Jose Nino (junr03) (jnino@lyft.com)
+* Daniel Hochman (danielhochman) (dhochman@lyft.com)
+* Michael Schore (goaway) (mschore@lyft.com)
+
```

Comment:
Are we doing to document the requirements for becoming a committer, or make these more informal?